### ISSUE TITLE
Github pages

### ISSUE THREAD

**Issue by tschuy** [iankronquist/how-to-train-your-compiler#1]
You should build the slides and stick them on the `gh-pages` branch so that you can link to them:

http://iankronquist.me/how-to-train-your-compiler

And then of course set the project's link to that.

**Comment by iankronquist** [comment]
Good idea and ridiculously simple to set up. I just created the branch and made a symlink from `index.html` to `build/sides/index.html`. Hopefully that will work.

**Comment by iankronquist** [comment]
I waited for an hour and it didn't show up. Instead I built the slides in the current directory. We'll see if this works.

**Comment by tschuy** [comment]
Looks likke your HTML is still a symlink to the build directory. Otherwise, things appaer to work? http://iankronquist.me/how-to-train-your-compiler/requirements.txt
